backend/bom-bom-server/src/main/java/me/bombom/api/v1/session/service/SessionCleanupService.java

```diff
@@ -0,0 +1,83 @@
+package me.bombom.api.v1.session.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import me.bombom.api.v1.session.repository.SessionManagerRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * 세션 정리 서비스
+ * Spring Session 테이블에서 만료된 세션을 삭제
+ */
+@Slf4j
+@Service
+@RequiredArgsConstructor
+@Transactional(readOnly = true)
+public class SessionCleanupService {
+
+    private final SessionManagerRepository sessionManagerRepository;
+
+    /**
+     * 만료된 세션 정리
+     * EXPIRY_TIME이 현재 시간보다 이전인 세션들을 삭제
+
+     * @return 삭제된 세션 수
+     */
+    @Transactional
+    public int cleanupExpiredSessions() {
+        long currentTimeMillis = System.currentTimeMillis();
+        
+        int deletedCount = sessionManagerRepository.deleteExpiredSessions(currentTimeMillis);
+        
+        if (deletedCount > 0) {
+            log.info("만료된 세션 정리 완료 - 삭제된 세션 수: {}", deletedCount);
+        } else {
+            log.debug("정리할 만료된 세션이 없습니다");
+        }
+
+        return deletedCount;
+    }
+
+    /**
+     * 철저한 세션 정리
+     * 만료된 세션뿐만 아니라 오래된 세션들도 함께 정리
+     * 
+     * @return 삭제된 세션 수
+     */
+    @Transactional
+    public int cleanupExpiredSessionsCompletely() {
+        long currentTimeMillis = System.currentTimeMillis();
+        // 추가로 30일 이상 된 세션도 정리
+        long thirtyDaysAgoMillis = currentTimeMillis - 30L * 24 * 60 * 60 * 1000;
```

Comment:
p4

이렇게 작성하면 더 직관적일 것 같습니다 😁
```suggestion
        long thirtyDaysAgoMillis = currentTimeMillis - Duration.ofDays(30).toMillis();
```

Reply:
오 이 방식이 더 좋은 것 같아요 
감사합니다!!

[refactor: 하드코딩 대신 유틸 메서드를 사용하도록 변경](https://github.com/woowacourse-teams/2025-bom-bom/pull/426/commits/6ed9cf57f8225b211854fd4d1cdd1409d9011b8c)